### DISCUSSION TITLE
TommyPROM code ported to Ben Eaters design.

### DISCUSSION THREAD

**Discussion by TheEccentricGenius** [TomNisbet/TommyPROM#58] [General]
I have successfully converted the TommyPROM code to work on Ben Eaters  design.
I have renamed it and licensed it under GPLv3.0, still need to do necessary changes for that.
I have given the credit to you in all the files also....

Currently only 28Cxx code is fully debugged and working, 27Cxx should work....
Everything offered by TommyPROM is supported, its just slower.

Here is the repo [EccentricPROM](https://github.com/TheEccentricGenius/EccentricPROM).

**Comment by TomNisbet** [comment]
Congratulations on making a version of TommyPROM to run on the Ben Eater boards.  People who are trying to burn code other than Ben's hard-coded sketches may find this useful.

I'm not sure I understand where you are going with this project beyond that first goal.  If you modify Ben's design to add high voltage programming or move the CE or OE signals, then you've now created a third hardware variant. What does this do that can't already be done with Ben's programmer or TommyPROM?  It seems confusing.

**Reply by TheEccentricGenius** [reply]
I think I will stick with the chips pin compatible with the 28Cxx....I'll  edit the github repo sometime to reflect this.

I did this conversion because your software is excellent....Working on it being able to program chips with  S-Records.
_P.S. I changed over to the MIT License._
